src/inmanta/const.py

```diff
@@ -238,3 +238,8 @@ class ParameterSource(str, Enum):
     plugin = "plugin"
     user = "user"
     report = "report"
+
+
+class ApiDocsFormat(str, Enum):
+    openapi = "openapi"
```

Comment:
How should we document the values so that the API docs will say what openapi and swagger is?